### PR TITLE
adds maintainer

### PR DESCRIPTION
**Pull request by jlaura**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jlaura** [comment]
@conda-forge-admin, please rerender

**Comment by dharvey-consbio** [comment]
Looks like `recipe/meta.yml` was updated, but not the `README`. Otherwise looks good to me.

**Comment by jlaura** [comment]
The tooling should have the README updated. Checkout the README.md file modified by this PR. Looks like you are being added properly?

File: README.md [hunk 1]
```diff
@@ -116,5 +116,6 @@ In order to produce a uniquely identifiable distribution:
 Feedstock Maintainers
 =====================
 
+* [@dharvey-consbio](https://github.com/dharvey-consbio/)
```
Comment (jlaura) [inline README.md:119]: @dharvey-consbio Here is the section I am referencing.

Comment (jlaura) [reply README.md:119]: I think the re-render added it?

**Comment by dharvey-consbio** [comment]
> Here is the section I am referencing.

Ah, I didn't see that in the commit history, so I missed it. Thanks!
image

**Pull request merged**